mantidimaging/gui/windows/spectrum_viewer/spectrum_widget.py

```diff
@@ -166,12 +185,9 @@ def colour_generator(self) -> tuple[int, int, int, int]:
     def change_roi_colour(self, name: str, colour: tuple[int, int, int, int]) -> None:
         """
         Change the colour of an existing ROI
-
-        @param name: The name of the ROI.
-        @param colour: The new colour of the ROI.
         """
-        self.roi_dict[name].colour = colour
-        self.roi_dict[name].setPen(self.roi_dict[name].colour)
+        roi = self.roi_dict[name]
+        roi.colour = colour
```

Comment:
I'd be tempted to make this one line
```
self.roi_dict[name].colour = colour
```
Also, can you keep the existing comments.